index.js

```diff
@@ -107,4 +107,15 @@ you have no test files or your tests are not generating any coverage data.
   return true;
 }
 
+function canUseMinCoverage(minCoverage) {
+  if (typeof minCoverage == 'number') return true;
+
+  if (minCoverage.toString().includes('%')) {
+    core.setFailed('❌ Failed to use min_coverage remove the `%` symbol');
+    return false;
+  }
+  core.setFailed('❌ Failed to use min_coverage value make sure you added a number');
+  return false;
```

Comment:
Should we not just parse the value as a int and see if we can? Instead of having "checks" for different cases

Reply:
@wolfenrain I don't get your point!